src/utils/misc.ts

```diff
@@ -100,6 +100,11 @@ export function pluralise(amount: number, singular: string, plural = singular +
     return amount === 1 ? `${amount} ${singular}` : `${amount} ${plural}`;
 }
 
+export function interpolateIfDefined(strings: TemplateStringsArray, ...args: any[]) {
+    if (args.some(arg => arg == null)) return "";
+    return strings.reduce((acc, str, i) => `${acc}${str}${args[i] ?? ""}`, "");
```

Comment:
this should work and be better

```suggestion
    return String.raw({ raw: strings }, ...args);
```